src/orac.F90

```diff
@@ -328,7 +328,7 @@ subroutine orac(mytask,ntasks,lower_bound,upper_bound,drifile)
    ! parameters and read the SAD values.
    allocate(SAD_Chan(Ctrl%Ind%Ny))
 
-   call Read_SAD(Ctrl, SAD_Chan, SAD_LUT)
+   call Read_SAD(Ctrl, global_atts%platform, SAD_Chan, SAD_LUT)
```

Comment:
I've reverted this change in my update to this PR (see https://github.com/ORAC-CC/orac/pull/61). In my update I add the netcdf LUT filename to the `Ctrl` struct, so the platform doesn't need to be parsed here.